cli/src/main/kotlin/StartCommand.kt

```diff
@@ -0,0 +1,100 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.cli
+
+import com.github.ajalt.clikt.command.SuspendingCliktCommand
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.long
+
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+import kotlinx.coroutines.delay
+import kotlinx.serialization.encodeToString
+
+import org.eclipse.apoapsis.ortserver.api.v1.model.CreateOrtRun
+import org.eclipse.apoapsis.ortserver.api.v1.model.JobConfigurations
+import org.eclipse.apoapsis.ortserver.api.v1.model.OrtRun
+import org.eclipse.apoapsis.ortserver.api.v1.model.OrtRunStatus
+import org.eclipse.apoapsis.ortserver.client.OrtServerClient
+
+internal val POLL_INTERVAL = System.getProperty("POLL_INTERVAL")?.toLongOrNull()?.milliseconds ?: 60.seconds
+
+class StartCommand(private val config: OrtServerOptions) : SuspendingCliktCommand(name = "start") {
+    private val repositoryId by option(
+        "--repository-id",
+        envvar = "REPOSITORY_ID",
+        help = "The ID of the repository."
+    ).long().required()
+
+    private val revision by option(
+        "--vcs-revision",
+        envvar = "VCS_REVISION",
+        help = "Revision of the repository to start the run for."
+    ).required()
+
+    private val path by option(
+        "--vcs-sub-path",
+        envvar = "VCS_SUB_PATH",
+        help = "Optional VCS sub-path of the repository."
+    )
+
+    private val jobConfigContext by option(
+        "--job-config-context",
+        envvar = "JOB_CONFIG_CONTEXT",
+        help = "Configuration context to use for the run."
+    )
+
+    private val wait by option(
+        "--wait",
+        envvar = "WAIT",
+        help = "Wait for the run to finish."
+    ).flag()
+
+    override suspend fun run() {
+        val client = OrtServerClient.create(config.toOrtServerClientConfig())
+
+        var ortRun = client.repositories.createOrtRun(
+            repositoryId = repositoryId,
+            ortRun = CreateOrtRun(
+                revision = revision,
+                path = path,
+                jobConfigs = JobConfigurations(),
+                jobConfigContext = jobConfigContext
+            )
+        )
+
+        if (wait) {
+            echo("Waiting for ORT run to finish...")
+
+            while (ortRun.isRunning()) {
+                delay(POLL_INTERVAL)
+                ortRun = client.repositories.getOrtRun(repositoryId, ortRun.index)
+            }
+        }
+
+        echo(json.encodeToString(ortRun))
```

Comment:
The idea behind using JSON is that the output should resemble the API response which is also in JSON. Also, the auto-generated `toString()` is fine for small objects but barely readable for larger nested objects because it does not use any line breaks.

Reply:
> the output should resemble the API response which is also in JSON.

But why's that beneficial? Would you copy & paste the response to Swagger or so?

Reply:
For consistency, because the API language is JSON. So it makes sense to me that the CLI output uses the same language.

Reply:
We have customers who want to parse the response to get the build ID and later do additional requests in their CI. Therefore, it would probably be good to a JSON-only output mode.